tools/ci_build/build.py

```diff
@@ -324,6 +324,9 @@ def parse_arguments():
     parser.add_argument(
         "--disable_wasm_exception_catching", action='store_true',
         help="Disable exception catching in WebAssembly.")
+    parser.add_argument(
```

Comment:
it looks like we can build either `onnxruntime_wasm.wasm` or `onnxruntime_wasm_threads.wasm`, but cannot build both. Is it possible to make them different targets so that we can build both of them in one build?